test/case/ietf_system/add_delete_user.py

```diff
@@ -23,8 +31,7 @@ def generate_restrictred_credential():
     with test.step("Add new user"):
         username = generate_restrictred_credential()
         password = generate_restrictred_credential()
-        salt = crypt.mksalt(crypt.METHOD_SHA256)
-        hashed_password = crypt.crypt(password, salt)
+        hashed_password = generate_password_hash(password, type="sha-256")
```

Comment:
Sorry, just a final comment.  This works, but only because `mkpasswd` parses `sha` and falls back to `sha256`.  The possible args are: `md5`, `sha256`, and `sha512`.  You can just fix the arg, and put the rest as a function description, within """...""".

Reply:
I tried sha256 but the function did not generate hash at all. I think proper syntax needs dash "-" between
![image](https://github.com/kernelkit/infix/assets/129860159/e9b359ce-8174-479b-a716-9b4963785a6e)

Reply:
It's the unfortunate difference between the standard Linux (Debian/Ubuntu) mkpasswd and the one shipped in Alpine Linux, which is built around the BusyBox family of tools:
```
$ make test-qeneth-sh
make[1]: Entering directory '/home/jocke/src/infix/buildroot'
/home/jocke/src/infix/test/env -f /home/jocke/src/infix/output/images/infix-x86_64.img -p /home/jocke/src/infix/output/images/infix-x86_64.pkg -q /home/jocke/src/infix/test/virt/dual /bin/sh
Info: Generating topology
Info: Generating node YAML
Info: Generating executables
Info: Launching dut1
Info: Launching dut2
(.infix-test-venv) /home/jocke/src/infix/buildroot # mkpasswd -h
mkpasswd: unrecognized option: h
BusyBox v1.36.0 (2023-05-05 06:41:49 UTC) multi-call binary.

Usage: mkpasswd [-P FD] [-m TYPE] [-S SALT] [PASSWORD] [SALT]

Print crypt(3) hashed PASSWORD

	-P N	Read password from fd N
	-m TYPE	des,md5,sha256/512 (default sha512)
	-S SALT
(.infix-test-venv) /home/jocke/src/infix/buildroot # 
```